src/components/CallView/CallView.vue

```diff
@@ -30,26 +30,23 @@
 			:local-shared-data="localSharedData" />
 
 		<template v-else>
-			<EmptyCallView v-if="!remoteParticipantsCount && !screenSharingActive && !isGrid" :is-sidebar="isSidebar" />
+			<EmptyCallView v-if="!callParticipantModels.length && !screenSharingActive && !isGrid" :is-sidebar="isSidebar" />
 
 			<div id="videos">
 				<template v-if="!isGrid">
 					<!-- Selected video override mode -->
 					<div v-if="showSelectedVideo"
 						class="video__promoted selected-video"
 						:class="{'full-page': isOneToOne}">
-						<template v-for="callParticipantModel in reversedCallParticipantModels">
-							<VideoVue v-if="callParticipantModel.attributes.peerId === selectedVideoPeerId"
-								:key="callParticipantModel.attributes.selectedVideoPeerId"
-								:token="token"
-								:model="callParticipantModel"
-								:shared-data="sharedDatas[selectedVideoPeerId]"
-								:show-talking-highlight="false"
-								:is-grid="true"
-								:is-big="true"
-								:is-one-to-one="isOneToOne"
-								:fit-video="true" />
-						</template>
+						<VideoVue v-if="selectedCallParticipantModel"
+							:token="token"
+							:model="selectedCallParticipantModel"
+							:shared-data="sharedDatas[selectedVideoPeerId]"
+							:show-talking-highlight="false"
+							:is-grid="true"
+							:is-big="true"
+							:is-one-to-one="isOneToOne"
+							:fit-video="true" />
 					</div>
 					<!-- Screens -->
 					<div v-else-if="showLocalScreen || showRemoteScreen || showSelectedScreen" id="screens">
```

Comment:
Please check if we used `id="screens"` somewhere. Otherwise, feel free to remove this div too

Reply:
only for styling, so replaced with a class. Still needed for Screen + presenter overlay components positioning though